### PR TITLE
Update install.go

### PR DESCRIPTION
**Pull request by gunjanmimo**

### CODE REVIEW

File: linkerd/install.go [hunk 1]
```diff
@@ -48,17 +48,17 @@ var (
...
 type APIInfo struct { //to solve
-	TagName    string   `json:"tag_name,omitempty"`//to solve
-	PreRelease bool     `json:"prerelease,omitempty"` //to solve
-	Assets     []*Asset `json:"assets,omitempty"` //to solve
-} //to solve
+	TagName    string   `json:"tag_name,omitempty"`
+	PreRelease bool     `json:"prerelease,omitempty"` 
+	Assets     []*Asset `json:"assets,omitempty"` 
+} 
+//to solve
```
Comment (girishranganathan) [inline linkerd/install.go]: Lets get rid of "to solve" and add a sensible comment here as well:
// Asset is used to store the individual asset data as part of a release

File: linkerd/install.go [hunk 2]
```diff
@@ -187,11 +187,12 @@ func (iClient *LinkerdClient) execute(command ...string) (string, string, error)
 		if os.IsNotExist(err) {
 			logrus.Error(err)
 			return "", "", err
-		} else {
-			err = errors.Wrap(err, "unknown error")
-			logrus.Error(err)
-			return "", "", err
 		}
+
```
Comment (girishranganathan) [inline linkerd/install.go]: This is good.

**Comment by girishranganathan** [comment]
The cause for failing builds in PRs #9 & #10 is this: https://github.com/layer5io/meshery-linkerd/pull/10/checks?check_run_id=223659621#step:4:121

looks like the struct definition was accidentally deleted.

**Comment by gunjanmimo** [comment]
sir  i am still facing errors

**Pull request merged**